### PR TITLE
add missing addShows template

### PR DESCRIPTION
**Pull request by OmgImAlexis**
Missed adding this back in the last vue PR.

### CODE REVIEW

**Comment by sharkykh** [comment]
Looks like you're giving the `gulp sync` command way too much credit... :sweat_smile:
My point is the files you renamed didn't get renamed in light and dark templates, just got duped.
Also with the previous PR you needed to edit the handler in the Python code. Don't you need to do that here? Pretty sure it won't work without that change.

**Comment by sharkykh** [comment]
Also need to fix `/addRecommended` (`addShows_recommended.mako`) with this PR.

**Comment by OmgImAlexis** [comment]
> Looks like you're giving the gulp sync command way too much credit... 😅

What do you mean?

**Comment by sharkykh** [comment]
>What do you mean?

I mean that it's not that smart.
If you rename or remove a file you need to manually do that for the dark and light themes.
As I said - 
> the files you renamed didn't get renamed in light and dark templates, just got duped.

image

**Comment by sharkykh** [comment]
Also:

image
image

File: themes-default/slim/static/js/templates/add-recommended.vue [hunk 1]
```diff
@@ -1,6 +1,5 @@
 <template>
     <div>
```
Comment (sharkykh) [inline themes-default/slim/static/js/templates/add-recommended.vue]: Now you don't need that wrapper div :)

File: themes-default/slim/static/js/templates/add-shows.vue [hunk 2]
```diff
@@ -1,6 +1,5 @@
 <template>
     <div>
```
Comment (sharkykh) [inline themes-default/slim/static/js/templates/add-shows.vue]: Same.

File: themes-default/slim/static/js/templates/add-recommended.vue (hunk 1 above)
Comment (OmgImAlexis) [reply themes-default/slim/static/js/templates/add-recommended.vue]: Didn't want to remove them incase we had any styles relying on the div depth. Haven't had a chance to check.

Comment (sharkykh) [reply themes-default/slim/static/js/templates/add-recommended.vue]: I did, it's okay, can be removed.
Keep the `<div id="addShowPortal">` though.

**Pull request merged**